recipe/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "recommenders" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/recommenders/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8db3785543b2ace6ddc68994e71c8920efa781432708d8d4cc7aff720f11b55f
+
+build:
+  number: 0
+  skip: True  # [py<38 or py>=310 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - bottleneck >=1.2.1,<2
+    - category_encoders >=1.3.0,<2
+    - cornac >=1.1.2,<2
+    - jinja2 >=2,<3.1
+    - lightfm >=1.15,<2
+    - lightgbm >=2.2.1
+    - matplotlib-base >=2.2.2,<4
+    - memory_profiler >=0.54.0,<1
+    - nltk >=3.4,<4
+    - numba >=0.38.1,<1
+    - pandas >1.0.3,<2
+    - pandera-core >=0.6.5
+    - python
+    - pyyaml >=5.4.1,<6
+    - requests >=2.0.0,<3
+    - retrying >=1.3.3
+    - scikit-learn >=0.22.1,<1.0.3
+    - scikit-surprise >=1.0.6
+    - scipy >=1.0.0,<2
+    - seaborn >=0.8.1,<1
+    - tqdm >=4.31.1,<5
+    - transformers >=2.5.0,<5
+    - numpy >=1.19
+
+test:
+  requires:
+    - pip
+  commands:
+    - pip check
+  imports:
+    - recommenders
+    - recommenders.datasets
+    - recommenders.evaluation
+    - recommenders.models
+    - recommenders.models.cornac
+    - recommenders.models.deeprec
+    - recommenders.models.deeprec.DataModel
+    - recommenders.models.deeprec.io
+    - recommenders.models.deeprec.models
+    - recommenders.models.deeprec.models.graphrec
+    - recommenders.models.deeprec.models.sequential
+    - recommenders.models.fastai
+    - recommenders.models.geoimc
+    - recommenders.models.lightfm
+    - recommenders.models.lightgbm
+    - recommenders.models.ncf
+    - recommenders.models.newsrec
+    - recommenders.models.newsrec.io
+    - recommenders.models.newsrec.models
+    - recommenders.models.rbm
+    - recommenders.models.rlrmc
+    - recommenders.models.sar
+    - recommenders.models.sasrec
+    - recommenders.models.surprise
+    - recommenders.models.tfidf
+    - recommenders.models.vae
+    - recommenders.models.vowpal_wabbit
+    - recommenders.models.wide_deep
+    - recommenders.tuning
+    - recommenders.tuning.nni
+    - recommenders.utils
+
+about:
+  home: https://github.com/microsoft/recommenders
+  license: MIT
+  license_family: MIT
+  license_url: https://github.com/microsoft/recommenders/blob/main/LICENSE
+  description: |
+    This package contains functions to simplify common tasks used when
+    developing and evaluating recommender systems.
+  summary: Microsoft Recommenders - Python utilities for building recommender systems
+  doc_url: https://pypi.org/project/recommenders/
```

Comment:
I think this should be the GH repo if a better suited page doesn't exist

Reply:
for the dev_url? or doc_url?